services/frontend/pkg/config/config.go

```diff
@@ -101,7 +101,7 @@ type Archiver struct {
 }
 
 type DataGateway struct {
-	Prefix string `yaml:"prefix" env:"FRONTEND_DATA_GATEWAY_PREFIX"`
+	Prefix string `yaml:"prefix" env:"FRONTEND_DATA_GATEWAY_PREFIX" desc:"Path prefix for the data gateway."`
```

Comment:
Is the prefix about the GATEWAY service?
`Path prefix for the data gateway.` --> `Path prefix for data to the GATEWAY service.`
It is just about consistent naming...

Reply:
No, its about the "data gateway" which is part of the FRONTEND service.

Basically it's one of our three api gateways:
- GATEWAY service acts as CS3 API gateway
- PROXY service acts as HTTP API gateway
- FRONTEND service (only the data gateway part of it) acts as up- and download (API) gateway